pkg/storage/azurecdn/storage.go

```diff
@@ -7,9 +7,8 @@ import (
 	"io"
 	"net/url"
 
-	"github.com/gomods/athens/pkg/config"
 	"github.com/gomods/athens/pkg/config/env"
-	multierror "github.com/hashicorp/go-multierror"
+	modupl "github.com/gomods/athens/pkg/storage/module"
```

Comment:
do you mind keeping this without the package alias? code in this file can then call `module.Thing` that way and it'll be a tiny bit clearer

same with the below `modupl` package alias

Reply:
@arschles this would cause conflicts with the variable name. I'm not entirely happy with the pkg name so maybe you have some better idea?

Reply:
Maybe just `mod`, I think it's the `upl` part that makes it less clear. In the past I've done things like `athensModule` or `athModule` etc. Thoughts @arschles ?

Reply:
mod would also cause conflicts